services/home/docs/demos/publisher-ad-quality-req.md

```diff
@@ -0,0 +1,224 @@
+Browser ->> Browser: Render ad
+```
+
+[Full-sized diagram](./img/publisher-ad-quality-req-flow.png)
+
+</TabItem>
+<TabItem value="demo" label="Demo">
+
+## Demo
+
+### Prerequisites
+
+- Chrome > v128 (Open chrome://version to look up your current version)
+
+### User Journey
+
+1. [Navigate to shop site](https://privacy-sandbox-demos-shop.dev/) (advertiser)
+2. Click on a “shoe” product item on the shop site. The shop (advertiser) would assume the user is interested in this type of shoe, so they would
+   leverage Protected Audience API and ask the browser to join an ad interest group for this product.
+3. [Navigate to the news site](https://privacy-sandbox-demos-news.dev/publisher-ad-quality-req) (publisher)
+4. Observe the ad served on the news site is for the shoe product you recently browsed.
+5. Click on a button which matches your selected shoe:
+   - Hide Red Shoes
+   - Hide Blue Shoes
+   - Hide Brown Shoes
+   - Hide Sports Shoes
+
+The page will refresh automatically and the shoe ad won’t display, this is because;
+
+- The button added the excludeCreativeTag parameter to the page url.
+- The auction requested the K/V value for the ad url from the SSP.
+- The SSP returned a list of product tags for this ad including product tag you wanted to exclude.
+- In the auction scoreAd function, the SSP marked this ad as having a score of 0 as it has a product tag which matches the one you wished to exclude
+  from the auction.
+
+### Implementation details
+
+#### How is the user added to an Interest Group based on his browsing behavior ? (see step #2 of User Journey)
+
+To add the user to an Interest Group, we reuse the implementation from the Retargeting / Remarketing use case, which is explained in
+_[Retargeting / Remarketing](https://privacy-sandbox-demos.dev/docs/demos/retargeting-remarketing) > Demo > How is the user added to an Interest Group
+based on his browsing behavior?_ Adding the browser to an interest group for a particular shoe allows us to render a winning ad in the shop site, or
+hide that ad based on the excludeCreativeTag page parameter.
+
+#### How do we serve an ad relevant to the user’s interest, or not if it was part of an excluded product tag? (see step #4 - #5 of User Journey)
+
+It’s in this part of the use case differs from the Retargeting / Remarketing & it uses a different news page.
+
+##### News page & SSP tag
+
+The publisher-ad-quality-req news page
+[(code link)](https://github.com/privacysandbox/privacy-sandbox-demos/blob/dev/services/news/src/views/publisher-ad-quality-req.ejs) reads the
+excludeCreativeTag and includes it in the sellerSignalExcludeCreativeTag as part of its adUnits list
+[(code link)](https://github.com/privacysandbox/privacy-sandbox-demos/blob/dev/services/news/src/views/publisher-ad-quality-req.ejs#L39). The news
+page also includes the SSP tag
+[(code link)](https://github.com/privacysandbox/privacy-sandbox-demos/blob/unified/services/ad-tech/src/public/js/ssp/ssp-tag.js) this will;
```

Comment:
use GitHub permalink feature instead which point to the [file](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files) on a specific commit or [code](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet). The unified branch will eventually be deleted but the commit will stay.

Example : 
https://github.com/privacysandbox/privacy-sandbox-demos/blob/1f9c6448072d922aa191f5ea2182ec92ead2fada/services/ad-tech/src/public/js/ssp/ssp-tag.js

Reply:
Done.